packages/typespec-client-generator-core/src/http.ts

```diff
@@ -185,7 +185,7 @@ function getSdkHttpParameters(
         kind: "body",
         name,
         isGeneratedName: true,
-        serializedName: "body",
```

Comment:
well if we would like to keep this property, we should fallback to its body

Reply:
Should we just remove the `serializedName` property at all?

Reply:
we could not do that. for explicitly defined body, it might have different client name. e.g., `op test(@body request: string): void`. in previous swagger example, we will set the request example with `{"request": "test"}`.

Reply:
yes.